src/constants.js

```diff
@@ -0,0 +1,3 @@
+export const newMatchCount = 'New matches';
```

Comment:
Im not crazy about make a constants file for 2 variables, but if thats the route it should at least follow malwares file schema. Same file but slapped insrc/components/SigDetailsTable(or SigTable)/constants
Also a little weird but at least it follows some kind of schema